src/components/Borrowers.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import { DataTable } from 'primereact/datatable';
+import { Column } from 'primereact/column';
+import { Button } from 'primereact/button';
+import { InputText } from "primereact/inputtext";
+import { Dialog } from "primereact/dialog";
+import {Message} from 'primereact/message';
+export class Borrowers extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            first: 0,
+            displayDialog: false,
+            valid: false,
+            nameValid: true,
+            phoneValid: true
+        }
```

Comment:
You can remove having to bind "this" by using arrow functions instead of "function"

Reply:
I knew that but forgot it when I was coding for some reason, fixed now